tests/Platforms/SqlitePlatformTest.php

```diff
@@ -250,6 +255,32 @@ public function getGenerateUniqueIndexSql(): string
         return 'CREATE UNIQUE INDEX index_name ON test (test, test2)';
     }
 
+    public function testGeneratesIndexCreationSqlWithSchema(): void
+    {
+        $indexDef = new Index('i', ['a', 'b']);
+        self::assertSame(
+            'CREATE INDEX main.i ON mytable (a, b)',
+            $this->platform->getCreateIndexSQL($indexDef, 'main.mytable'),
+        );
```

Comment:
This method contains two test scenarios in a single test. Furthermore, this first scenario is green without your changes. It does not really cover anything.

Reply:
I have split the test into two.

The first test tests if expected SQL is generated when a table name with schema is given.

The second test tests if unmodified table/column is passed to `SqlitePlatform::getCreatePrimaryKeySQL()`.

Reply:
@derrabus does this explanation makes sense to you?

Reply:
@derrabus can I please get your approval here?

Reply:
@derrabus does this explanation makes sense to you?